gpu-operator/gpu-operator-kubevirt.rst

```diff
@@ -242,6 +242,20 @@ The following example shows how to permit the A10 GPU device and A10-24Q vGPU de
             - externalResourceProvider: true
               pciVendorSelector: 10DE:2236
               resourceName: nvidia.com/GA102GL_A10
+      ...
+
+   Add the following for vGPU:
+
+   .. code-block:: yaml
+
+      ...
+      spec:
+        configuration:
+          developerConfiguration:
+            featureGates:
+            - GPU
+            - DisableMDEVConfiguration
+          permittedHostDevices: # Defines VM devices to import.
             mediatedDevices: # Include for vGPU 
             - externalResourceProvider: true
               mdevNameSelector: NVIDIA A10-24Q
```

Comment:
Let's update the profile here to be consistent with the profile used earlier in the procedure
```suggestion
              mdevNameSelector: NVIDIA A10-12Q
```

Reply:
GitHub is not letting me add a suggestion for the proceeding line, but we also need to update the `resourceName` field to `nvidia.com/NVIDIA_A10-12Q`